README.md

```diff
@@ -62,7 +62,7 @@ custom_components
 
 [![ha_badge][ha-add-shield]][ha-add-url]
 
-To configure the component, add it using [Home Assistant integrations][ha-add-url]. This will provide you with a configuration screen where you can first select the data source. Again, most inverters use JS. Some use JSON and in some rare cases HTML is used.
+To configure the component, add it using [Home Assistant integrations][ha-add-url]. This will provide you with a configuration screen where you can first select the data source. Again, most inverters use JS. Some use JSON and in some rare cases HTML is used. The TCP backend contains additional electrical statistics but lacks information about the WiFi module.
```

Comment:
Will have to reword this some if/when we recommend TCP and/or a TCP+JS hybrid connection.